pkg/gui/popup/popup_handler.go

```diff
@@ -1,11 +1,12 @@
 package popup
 
 import (
+	"context"
 	"strings"
 
 	"github.com/jesseduffield/gocui"
 	"github.com/jesseduffield/lazygit/pkg/common"
-	"github.com/jesseduffield/lazygit/pkg/gui/context"
+	gctx "github.com/jesseduffield/lazygit/pkg/gui/context"
```

Comment:
sorry about the awkwardness here. I wanna rename our 'context' to 'panel' now that it's 1:1 with views